src/siteconfig/models.py

```diff
@@ -152,6 +155,12 @@ class SiteConfig(models.Model):
         help_text="Set up at least one Mark Range in admin for this to do anything."
     )
 
+    enable_google_signin = models.BooleanField(
+        verbose_name="Enable sign-in via Google",
+        default=False,
+        help_text="Used to enable or disable google sign-in"
```

Comment:
help_text is a repeat of the verbose name and redundant.  It should give useful info or remove.